### ISSUE TITLE
Error: require() of ES modules is not supported.

### ISSUE THREAD

**Issue by Xapu1337** [pmndrs/drei#450]
Hey I'm using `@react-three/fiber & drei` it worked in the beginning then I've got a massive error something like: 
```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: C:\Users\XapuR\OneDrive\xapu1337-website\node_modules\three-mesh-bvh\src\index.js
require() of ES modules is not supported.
require() of C:\Users\XapuR\OneDrive\xapu1337-website\node_modules\three-mesh-bvh\src\index.js from C:\Users\XapuR\OneDrive\xapu1337-website\node_modules\@react-three\drei\index.cjs.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES module
s.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from C:\Users\XapuR\OneDrive\xapu1337-website\node_modules\three-mesh-bvh\package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1080:13)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\Users\XapuR\OneDrive\xapu1337-website\node_modules\@react-three\drei\index.cjs.js:1:699)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14) {
  code: 'ERR_REQUIRE_ESM'
}
```
The source to this file can be found [here](https://github.com/Xapu1337/xapu1337-website/blob/master/pages/webgltest.tsx)

**Comment by drcmda** [comment]
imo there is no fix. this is coming from node and from other packages, in this case three-mesh-bvh. someone needs to tell them to change their package.

**Comment by joshuaellis** [comment]
Well the fix is to remove the feature until it's resolved in the library if anything.

**Comment by drcmda** [comment]
i agree, removing it is probably for the best.

**Comment by ragnorc** [comment]
> Well the fix is to remove the feature until it's resolved in the library if anything.

@joshuaellis What do you mean by "remove" the feature?

Is there any to fix this before it gets fixed in three-mesh-bvh?

**Comment by drcmda** [comment]
only by going back in version history. the fix should be easy for them, i hope this can be fixed quickly but no response yet.

**Comment by 3dln** [comment]
Same error here and this makes drei completely unusable, so makes sense to remove the feature until they fix the bug

**Comment by 3dln** [comment]
> 
> 
> only by going back in version history. the fix should be easy for them, i hope this can be fixed quickly but no response yet.

Which version is OK to use @drcmda?

**Comment by joshuaellis** [comment]
> > 
> 
> > 
> 
> > only by going back in version history. the fix should be easy for them, i hope this can be fixed quickly but no response yet.
> 
> 
> 
> Which version is OK to use @drcmda?

The last 0.X.0 version. We use semver so it's easy to understand.

**Comment by Xapu1337** [comment]
> > > 
> > 
> > 
> > > 
> > 
> > 
> > > only by going back in version history. the fix should be easy for them, i hope this can be fixed quickly but no response yet.
> > 
> > 
> > Which version is OK to use @drcmda?
> 
> The last 0.X.0 version. We use semver so it's easy to understand.

The last drei version?
Because if drei is currently v6.2.0 v0.X.0 sounds a bit outdated?
or maybe I'm wrong

**Comment by 3dln** [comment]
> 
> 
> > > > 
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > only by going back in version history. the fix should be easy for them, i hope this can be fixed quickly but no response yet.
> > > 
> > > 
> > > Which version is OK to use @drcmda?
> > 
> > 
> > The last 0.X.0 version. We use semver so it's easy to understand.
> 
> The last drei version?
> Because if drei is currently v6.2.0 v0.X.0 sounds a bit outdated?
> or maybe I'm wrong

I uses 6.1.0 and the error is gone

**Comment by joshuaellis** [comment]
I didn't expect people to take is so literally and actually believe I was suggested v0. But good to know. I'm locking this conversation for maintainers as we've identified, the problem, a work around and we'll be going from there.

**Comment by joshuaellis** [comment]
this has been released in v6.2.3-beta.1 if you could test it @Xapu1337 that would be great!
